<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgUserPasswordChange</class>
 <widget class="QDialog" name="DlgUserPasswordChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>126</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>328</width>
    <height>126</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>328</width>
    <height>126</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zmiana hasła</string>
  </property>
  <widget class="QPushButton" name="btnChange">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zmień</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Dotychczasowe hasło:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtOldPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Nowe hasło</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNewPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>311</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Powtórz nowe hasło:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNewPasswordRep">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
